crawler/crawler/cloneRepository.go

```diff
@@ -33,17 +33,13 @@ func CloneRepository(domain Domain, hostname, name, gitURL, gitBranch, index str
 		if err != nil {
 			return errors.New(fmt.Sprintf("cannot git pull the repository: %s: %s", err.Error(), out))
 		}
-		// Command is: git reset --hard origin/<branch_name>
-		out, err = exec.Command("git", "-C", path, "reset", "--hard", "origin/"+gitBranch).CombinedOutput() // nolint: gas
```

Comment:
Is wanted that we removed git reset --hard?

Reply:
Uhm, actually we want it in. I didn't notice we are `git fetch`ing instead of `git pull`ing a few lines above (we'll probably take care of that for #249). Thanks!

Let's `reset` to `origin/HEAD`.